corehq/apps/users/templates/users/partial/basic_info_modals.b3.html

```diff
@@ -0,0 +1,62 @@
+{% load crispy_forms_tags %}
```

Comment:
```basic_info_modals.html``` seems to be used in other places so I've created new one. I'm not sure if it's correct approach.